vancouver-tree-height-geo.Rmd

```diff
@@ -168,7 +168,13 @@ sd
 
 ## Discussion
 
-sd
+From the plot, we observed the RENFREW-COLLINGWOOD neighborhood has the darkest color among others indicating the highest concentration of trees for the shortest trees, following by neighborhood like HASTINGS-SUNRISE, KENSINGTON-CEDAR COTTAGE, VICTORIA-FRASERVIEW and SUNSET.
```

Comment:
Maybe we can firstly address the main statistical question we are interested in: Tree heights do vary significantly from neighborhood to neighborhood.

Reply:
Conclusion added.